jormungandr-integration-tests/src/common/jormungandr/starter.rs

```diff
@@ -181,9 +189,22 @@ impl Starter {
         self.config.as_ref().unwrap().clone()
     }
 
+    pub fn start_benchmark_run(&self) -> Option<SpeedBenchmarkRun> {
```

Comment:
No plans to use the `#[bench]` infrastructure?
We could build with nightly for benchmarks, it's not a big problem now.

Reply:
noted down. will explore this possibility one in the nearest free time